src/features/auth/services/login.service.ts

```diff
@@ -3,11 +3,11 @@ import { BackendError } from "app/exceptions";
 import axios from "axios";
 import { Role } from "core/users/types";
 
-const URL = `${BACKEND_V1_URL}/auth/login`;
+const BASE_URL = `${BACKEND_V1_URL}/auth/login`;
 
 export default async function login(body: LoginBody): Promise<LoginResponse> {
   try {
-    const response = await axios.post(URL, body);
+    const response = await axios.post(BASE_URL, body);
```

Comment:
Actually it is a `URL` not a `BASE_URL`, `BASE_URL` is `BACKEND_V1_URL`.

Reply:
de hecho, eso cambia con el .env, esta resolvido.

Reply:
PERO YA VA, tu comentario es literal todo lo contrario a tu sugerencia xd